.github/workflows/pkgdown.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
             BASE_IMAGE=${{ env.BASE_IMAGE }}
 
       - name: Run pkgdown
-        if: ${{ inputs.IS_SUBDIR  == 'false' }}
+        if: ${{ inputs.IS_SUBDIR  != 'true' }}
```

Comment:
Just out of curiosity - what is the reason for this change?

Reply:
It's required to properly support gDRstyle case ('IS_SUBDIR' not defined and equals '').